lnrpc/walletrpc/walletkit.proto

```diff
@@ -839,6 +843,9 @@ message EstimateFeeResponse {
     confirmation target in the request.
     */
     int64 sat_per_kw = 1;
+
+    // The current minimum relay fee based on our chain backend in sat/kw.
+    SatPerKw min_relay_fee = 2;
```

Comment:
I get your idea with wanting to add more context with a sub message.
But I don't think this would come through nicely, both in gRPC and REST:
```
"min_relay_fee":  {
        "rate":  "253"
    }
```

So I think it should either be `rate_sat_per_kw` in the new `SatPerKw` message. Or just `int64 min_relay_fee_sat_per_kw = 2` here.

Also, we convert things in the CLI, so the new field needs to be added here: https://github.com/lightningnetwork/lnd/blob/fc90bc9b0f2b4238ea28a33eeb5b63394454701b/cmd/lncli/walletrpc_active.go#L168

Reply:
Good point, will go with the old suffix route. I do wish `proto3` supported type aliases though...